src/lib/SingleFileDB.hpp

```diff
@@ -30,17 +30,25 @@ class SingleFileDB
 public:
     explicit
     SingleFileDB(
-        llvm::StringRef pathName)
+        llvm::StringRef pathName, bool is_clang_cl = false)
     {
         auto fileName = files::getFileName(pathName);
         auto parentDir = files::getParentDir(pathName);
 
         std::vector<std::string> cmds;
-        cmds.emplace_back("clang");
+        if (is_clang_cl) {
+            cmds.emplace_back("clang-cl");
```

Comment:
This path is unnecessary. is_clang_cl would be generating behavior that's externally functionally equivalent for both true and false.

Reply:
I understand that. Besides the obvious problem that clang-cl is not the only compiler on Windows, that's not the role of this class, and everything else I said above. This is not a test class, and you're including a change the class doesn't need. I left another comment explaining how you can test the change you actually want to test.